config/configmap/atlas-dbaas-registry.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mongodb-atlas-registration
+  labels:
+    related-to: dbaas-operator
+    type: dbaas-provider-registration
+data:
+  provider: MongoDBAtlas
+  inventory_kind: MongoDBAtlasInventory
+  connection_kind: MongoDBAtlasConnection
+  credentials_fields: |
+    - key: orgId
+      display_name: Organization ID
+      type: string
+      required: true
+    - key: publicApiKey
+      display_name: Public API Key
+      type: maskedstring
```

Comment:
Should this be masked?

Reply:
good question. Public keys do not need to be masked so I updated it. Thanks.